contents/tutorials/nextjs-cookie-banner.md

```diff
@@ -70,17 +81,13 @@ Importantly, to avoid a hydration error, we must check if the frontend has mount
 'use client';
 import { useEffect, useState } from "react";
 
-export default function Banner() {
+export function Banner() {
   const [showBanner, setShowBanner] = useState(false);
```

Comment:
just set the initial state to true for now? Then we can remove the bit about the hydration error

Reply:
As far as I remember, the hydration error happens if you don't use `useState` at all.